### DISCUSSION TITLE
ODFs plot in MTEX is different to ODFs plot in Channel 5

### DISCUSSION THREAD

**Discussion by kopparthiravikiran** [mtex-toolbox/mtex#1774] [Ask Anything]
Hello,

I have plotted ODFs in both MTEX and Channel 5, and it seems both plots are different. Please find the attached plots. I think I am doing a mistake in choosing either specimen or crystal symmetry in MTEX. Could anyone let me know what I am doing wrong. I have tried changing the plot conventions and symmetries but it's not working for me.

I used the below code in Matlab to plot ODFs:

%% Import Script for EBSD Data
%
% This script was automatically created by the import wizard. You should
% run the whoole script or parts of it in order to import your data. There
% is no problem in making any changes to this script.

%% Specify Crystal and Specimen Symmetries

% crystal symmetry
CS = {...
  'notIndexed',...
  crystalSymmetry('m-3m', [2.9 2.9 2.9], 'mineral', 'Iron-alpha'),...
  crystalSymmetry('m-3m', [3.7 3.7 3.7], 'mineral', 'Iron'),...
  'notIndexed',...
  'notIndexed'};

% plotting convention
setMTEXpref('xAxisDirection','east');
setMTEXpref('zAxisDirection','outOfPlane');

%% Specify File Names

% path to files
pname = 'C:\Users\New Owner\Desktop';

% which files to be imported
fname = [pname '\1.cpr'];

%% Import the Data

% create an EBSD variable containing the data
ebsd = EBSD.load(fname,CS,'interface','crc',...
  'convertEuler2SpatialReferenceFrame');
%%
odf.SS = specimenSymmetry('222');

plot(odf,'sections',18,'layout',[5 4],...
  'coordinates','off','xlabel','','ylabel','')

![2](https://user-images.githubusercontent.com/128254603/226196960-70847eac-8b26-46d2-bd6c-8a747fbcbd08.JPG)

**Comment by kilir** [comment]
Hi,
a) are you sure you are plotting the coresponding phases?
b) it makes a difference whether you symmetrise the orientations first and compute an odf or if you compute an odf set the specimenSymmetry of yout odf afterwards - the latter is what you are doing.
Cheers,
Rüdiger

**Reply by kopparthiravikiran** [reply]
Hello,

I am plotting for the 'Iron-alpha' phase. Could you please elaborate on the symmetrise orientations first and compute ODF later?

%%
orientation=ebsd('Iron-alpha').orientations

odf=calcDensity(orientation)

odf.SS = specimenSymmetry('222');

plot(odf,'sections',18,'layout',[5 4],...
'coordinates','off','xlabel','','ylabel','')

Thanks,
Ravi.
